packages/firestore/src/lite-api/database.ts

```diff
@@ -41,12 +41,15 @@ import { cast } from '../util/input_validation';
 import { logWarn } from '../util/log';
 
 import { FirestoreService, removeComponents } from './components';
+import type { PipelineSource } from './pipeline-source';
 import {
   DEFAULT_HOST,
   FirestoreSettingsImpl,
   PrivateSettings,
   FirestoreSettings
 } from './settings';
+// `import type` to avoid bundling the source for
+// pipelines if `useFirestorePipelines()` is not called
```

Comment:
I think this comment must have been for line 44 (but auto formatting may have moved that code?)